projects/demo/src/modules/app/utils/sort-pages.ts

```diff
@@ -0,0 +1,32 @@
+import {isPageGroup, TuiDocPage, TuiDocPageGroup} from '@taiga-ui/addon-doc';
+
+export const sortPages = <T extends TuiDocPage | TuiDocPageGroup>(
+    pages: readonly T[],
+    excludeSections: string[],
```

Comment:
where do we use `excludeSections` for your sorting algorithm?

Reply:
sorry, I forgot it